AppHandling/Run-AlPipeline.ps1

```diff
@@ -506,7 +508,7 @@ function GetInstalledAppIds {
     Write-GroupStart -Message "Installed Apps"
     $installedApps | ForEach-Object {
         Write-Host "- $($_.AppId):$($_.Name)"
-        "$($_.AppId)"
+        return @{ "Id" = "$($_.AppId)"; "Name" = "$($_.Name)"; "Publisher" = "$($_.Publisher)"; "Version" = "$($_.Version)" }

```

Comment:
The code in GetInstalledAppIds adds the field "AppId", but this code here uses "Id". Align?

Reply:
There are 3 different ways to get the installed apps - and they return different objects - this function will return this in the way needed by the rest of containerhelper and AL-Go, a hashtable with id, name, version and publisher.